src/test/java/integration/BaseTest.java

```diff
@@ -47,7 +47,7 @@ public static void setUp() {
     }
 
     public static CompletableFuture<Entry> createEntry(RepositoryApiClient client, String entryName,
-            Integer parentEntryId, Boolean autoRename) {
+                                                       Integer parentEntryId, Boolean autoRename) {
```

Comment:
I think the original form is better as it is the form what IDEs usually follow. No need to manually format the code.